canonical-web-and-design/maas-ui
# Serve UI only

## Done

- Update the scripts and proxy to only serve the React client.
- Include Vanilla in the React client.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Run `yarn start`.
- The UI should be available at the normal url you visit (with port 8400, not the port displayed by the React server).
- Have a look around and make sure the styles don't look broken.

## Fixes

Fixes: canonical-web-and-design/app-tribe#953.
Fixes; canonical-web-and-design/app-tribe#1021.
